src/models/sqlModels.ts

```diff
@@ -0,0 +1,33 @@
+import { DataTypes } from 'sequelize';
+const instances = require('hapi-sequelizejs').instances;
+
+module.exports.taskModel = function () {
```

Comment:
Oh, okay so is sqlModels.ts really rendering all of models.ts obsolete?

Reply:
Yeah, Kyle just wanted me to keep it in the repo for some reason